.github/CODEOWNERS

```diff
@@ -20,10 +20,10 @@
 /LICENSE @nodejs/tsc
 /onboarding.md @nodejs/tsc
 
-# website
-/doc/api_assets @nodejs/website
-/doc/template.html @nodejs/website
-/tools/doc @nodejs/website
+# nodejs.org website
+/doc/api_assets @nodejs/nodejs-website
+/doc/template.html @nodejs/nodejs-website
+/tools/doc @nodejs/nodejs-website
```

Comment:
@Trott could you update this one for `@nodejs/web-infra`?

Reply:
```suggestion
/tools/doc @nodejs/web-infra
```